src/expression.js

```diff
@@ -83,15 +99,7 @@ Bracket.prototype.value = function (scope) {
 	if (!obj) {
 		return lookupValue(prop, scope, {}, {}).value;
 	} else {
-		return compute(function() {
-			if (obj instanceof Lookup) {
-				obj = lookupValue(obj.key, scope, {}, {}).value();
```

Comment:
This was probably creating a compute w/i another compute which isn't typically optimal and can be avoided.